server/src/test/java/org/apache/druid/server/coordinator/simulate/SegmentBalancingTest.java

```diff
@@ -296,7 +296,7 @@ public void testMaxSegmentsAreMovedWhenClusterIsSkewed()
     runCoordinatorCycle();
     verifyValue(Metric.ASSIGNED_COUNT, 10_000L);
     verifyNotEmitted(Metric.MOVED_COUNT);
-    verifyValue(Metric.MOVE_SKIPPED, 100L);
+    verifySum(Metric.MOVE_SKIPPED, 100L);
```

Comment:
Could you please explain why this needed to change for this patch ?

Reply:
The metric is now being emitted for multiple servers. So we need to verify the overall sum value.